test/PerformanceTests/SerializationComparisonsTests/JsonWriterBenchmarks/Benchmarks.cs

```diff
@@ -65,29 +71,30 @@ public async Task WriteToMemoryAsync()
         public async Task WriteToFileAsync()
         {
             // multiple writes to increase benchmark duration

```

Comment:
This comment needs to be updated also?

Reply:
Two writes are still multiple. I think the intent behind the comment still stands.